exercises/grep/grep.js

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
+
+// The above line is a shebang. On Unix-like operating systems, or environments, this will allow the script to be
+// run by node, and thus turn this JavaScript file into an executable. If you don't have a Unix-like operating
+// system or environment, for example Windows without WSL, you can use 
+//
+// node grep.js args
+//
+// Instead of "grep args".
```

Comment:
Shouldn't that be more like `"./grep.js args"`?

The file is still named with suffix `.js` and if it is not added to the `$PATH` I don't think just writing the name will execute it (at least not in my shell).

Reply:
Indeed

Reply:
The POSIX variant is `grep.js args` if you're in the right directory, but this comment was initially meant to mimic `actual-grep args`